src/Core/Localization/Pack/Factory/LocalizationPackFactoryInterface.php

```diff
@@ -24,17 +24,18 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
-namespace PrestaShop\PrestaShop\Core\Localization\Pack;
+namespace PrestaShop\PrestaShop\Core\Localization\Pack\Factory;
+use LocalizationPack;
```

Comment:
you cant' use object model directly in Core classes. Rely on an Adapter instead.

Reply:
yup. :) I found that there's adapter aliases for legacy object models under `PrestaShop\PrestaShop\Adapter\Entity` namespace, is it okay if i use it?

Reply:
Yes, thanks